pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if [ ! -z "$DOCKER_HOST" ] && (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
Suggestion:

```bash
[[ "$OSTYPE" =~ 'darwin' ]]
```

(And if you feel like cleanning up Bash, we could also use that in [line 20](https://github.com/openshift/source-to-image/pull/586/files#diff-1a14ec8a07d06e9da1a51072db9046bfR20), and other occurrences if any)